### ISSUE TITLE
Google login is not working

### ISSUE THREAD

**Issue by gigalala** [jabbink/PokemonGoBot#545]
Hi,
Bot was working fine all day but when i tried to start it again i got:

```
D/GoogleCredentialProvider: 0
D/GoogleCredentialProvider: Get user to go to:null and enter code:null
```

**Comment by Agronis** [comment]
If you haven't tried already, remove token from config file and try again.

**Comment by Mewks** [comment]
This happened to me. Just try a few times, after my 3rd time opening the bot, it worked fine.

**Comment by gigalala** [comment]
Tried 30 times by now, still not working!

**Comment by pvase666** [comment]
Fill exactly your username and password.

**Comment by chandler14362** [comment]
You probably got banned.

**Comment by mozzy306** [comment]
same issue it works on 3 acc but 1 has this issue

**Comment by gigalala** [comment]
i tried some other account and i still have the same issue

**Comment by gigalala** [comment]
I don't think i am banned because i can log in with my phone

**Comment by forkrave** [comment]
This issue occurs whenever you get banned. Your account is now banned from their servers.

**Comment by gigalala** [comment]
Still doesn't explain why the problem also occur with the other account (new)  i have, or how i can still open it on my phone.

**Comment by Coldasicex** [comment]
I have a similar problem 
D/GoogleCredentialProvider: 0
Server refused your login credentials. Are they correct? [0m
And I just checked on Nox, my account is not even softbanned
Running on WX 64x

**Comment by Necromoni** [comment]
Recompile the bot and try again

**Comment by Mewks** [comment]
Strangely enough same thing is happening to me now. Was working fine before. Using the release jar.

**Comment by Mewks** [comment]
![image](https://cloud.githubusercontent.com/assets/20590397/17201786/1a8a3884-54d5-11e6-8ee5-9abe15def241.png)

Login server is down thats why.

**Comment by okynk** [comment]
i can't login via google too..
i already make sure that the token field is empty and got same result as others
```
28 Jul 05:38:04: Logging in to game server...
D/GoogleCredentialProvider: 0
D/GoogleCredentialProvider: Get user to go to:null and enter code:null
```

@Mewks can you share where you got that info?

**Comment by Mewks** [comment]
Hopefully I'm allowed to post links in here?

http://ispokemongodownornot.com/

**Comment by langerhans** [comment]
Google login seems to be back up, can you try again? Make sure you only configure mail address and password, no token.

**Comment by Mewks** [comment]
Still happening, tried multiple accounts.

**Comment by Coldasicex** [comment]
tried without token, got:
Logging in to game server...
D/GoogleCredentialProvider: 0
D/GoogleCredentialProvider: Get user to go to:null and enter code:null

**Comment by Mewks** [comment]
Yeah seems to be a common issue right now. All my PTC logins are working fine.

**Comment by Hamish02** [comment]
Same problem :/

**Comment by nekolasin** [comment]
same problem but in mobil phone works nice

**Comment by Coldasicex** [comment]
>  nekolasin commented a minute ago
>same problem but in mobil phone works nice

Yeah, in Nox is working too

**Comment by Mewks** [comment]
Maybe they are shutting down the Google API access and keeping PTC open for now?

**Comment by Coldasicex** [comment]
> Maybe they are shutting down the Google API access and keeping PTC open for now?

bot is working with PTC?

**Comment by Mewks** [comment]
@Coldasicex yep!

**Comment by okynk** [comment]
> Yeah seems to be a common issue right now. All my PTC logins are working fine.

same here..

**Comment by Skander1998** [comment]
Same issue.

**Comment by langerhans** [comment]
This seems to be a general issue with Google login and the java api, not specific to the bot. See Grover-c13/PokeGOAPI-Java#293 

I guess be patient for now. Maybe Google changed something about the flow or is actually selectively blocking things. We don't know yet.

**Comment by Grover-c13** [comment]
This seems to be an issue with Google/Niantic

**Comment by Mewks** [comment]
I don't really know how the API and stuff works so this might be a dumb statement but is it possible that Niantic are implementing some stuff into their API to perhaps detect our cheating?

**Comment by langerhans** [comment]
No one knows for certain yet. They have changed _something_ tho as the application name on the web oauth page now shows `Pokemon Go` instead of `Pokemon Go Release` as it was before.

**Comment by karlonovak** [comment]
Ok so the thing probably is that Niantic changed the Pokemon GO API Client ID and Secret key.
They immediately changed it in their original app, but someone has to decompile it now and get a new Client ID and Secret key that the app is now using. They can't stop you from using Google login 'cause it would be really hard from them to tell the original app from Bot.

**Comment by langerhans** [comment]
No new app has been released recently, also the web based oauth flow works just fine with the old client_id, my app can use Google login just fine. So that's not the case for sure.

**Comment by Mewks** [comment]
Wow everything still works fine for you @langerhans ?

**Comment by langerhans** [comment]
In my app yes, cause I'm not using the login flow provided by the library. But that is unrelated to the bot.

**Comment by Mewks** [comment]
OHH yeah I see what you mean. App works fine for me too. If what karlo said was the case, wouldn't that mean they would have to release an update for the app with the new clientid/secret? I guess we just wait while the smart cookies work on this!

**Comment by ryan786** [comment]
Same problem from today .Resolve the issue please

**Comment by Mewks** [comment]
Chill dude, they are working on it. @ryan786

**Comment by okynk** [comment]
the PokeGOAPI-Java lib already fixed & updated, now we just need to chill & wait the devs working on it..

**Comment by ryan786** [comment]
Yeah Just read about it that it was fixed ,now waiting for it to be implemented in this jar

**Comment by karael** [comment]
i m fixing it, PR incoming

**Comment by TunnelSnakesThatsus** [comment]
@karael whats pr?

**Comment by TunnelSnakesThatsus** [comment]
can anyone tell me what pr is??

**Comment by michaeldajnowicz** [comment]
pull request

**Comment by TunnelSnakesThatsus** [comment]
ok thanks!

**Comment by michaeldajnowicz** [comment]
@jabbink it would be great if this was fixed for all version if not at least 0.4.2 since its the most stable for me, thanks

**Comment by langerhans** [comment]
Locking this for now. We are aware of the problem and a fix is being worked on. This spam is not useful at all.

**Comment by langerhans** [comment]
Fixed by #575 . release incoming.
